### PR TITLE
Vxiong branch

### PR DESCRIPTION
**Pull request by victor-xiong**
npm install will do:
1. npm install
2. bower install

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +22,9 @@
     "url": "https://github.com/victor-xiong/oh-hell/issues"
   },
   "homepage": "https://github.com/victor-xiong/oh-hell#readme",
+  "dependencies": {
```
Comment (martinsuchen35) [inline package.json]: What's the difference between "dependencies" and "devDependencies"?
Does bower need version set in "dependencies"?

**Comment by victor-xiong** [comment]
http://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies

it looks like there would be no difference in our usage, so I deleted dependencies.

**Comment by victor-xiong** [comment]
added more packages required by testing

**Comment by martinsuchen35** [comment]
Looks good. Ship it!

**Pull request merged**